field_rrule/hooks.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import socket
+import inspect
+
+
+def post_load_hook():
+    """do some trickery to have demo data/model on runbot, but nowhere else"""
```

Comment:
Sorry, I misread. I thought you could check if demo mode was activated. This is both very clever and a horrible hack. What was wrong with the inclusion of a dedicated demo module again?